src/kirby/components/segmented-control/segmented-control.component.scss

```diff
@@ -42,6 +42,7 @@
     }
     
     kirby-badge {
+      --kirby-badge-elevation: #{get-elevation(2)};
```

Comment:
Consider outputting elevations as css variables and having `get-elevation(2)` return `var(--kirby-elevation-z2)` etc to avoid repetition